includes/misc-functions.php

```diff
@@ -998,3 +998,136 @@ function edd_doing_cron() {
 	// Default to false
 	return false;
 }
+
+/**
+ * SVG dimensions
+ * 
+ * @since 2.9.x
+ *
+ * @return array $sizes Sizes array (width and height) of the icon requested.
+ */
+function edd_svg_dimensions( $icon ) {
+	
+	if ( empty( $icon ) ) {
+		return false;
+	}
+	
+	$sizes = apply_filters( 'edd_svg_dimensions', 
```

Comment:
Note: Each SVG icon has its dimensions added to this array. This is so:

1. We don't need to set icon dimensions via CSS. Someone could add their own SVG icon, add their dimensions and be done with it.
2. Setting a width and height directly on the SVG is better IMO. If the browser has CSS disabled the icons will still be the same size. Set the size in CSS and when disabled the icons will take over the whole browser.